### PR TITLE
Added Careem in User list

### PR DESCRIPTION
**Pull request by jatin5251**
Added Careem in User list

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -92,6 +92,19 @@
...
+- name: Careem
+  anchor: Careem
+  logo: /assets/images/logos/careem.png
+  logosmall: /assets/images/logos/careem-small.png
+  testimonial: |
+    Trino, the distributed query engine for big data that uses the SQL query language,
+    became the tool of choice for ad hoc data analysis at Careem. Paired with dashboarding tools,
+    Presto is a valuable platform to quickly analyze data. Many data analysts and data scientists 
+    use Jupyter Notebooks consistently to write automated reports and schedule queries. 
+    This makes Presto one of the most critical services to serve as a backbone of data access
```
Comment (nineinchnick) [inline _data/users.yml:104]:
```suggestion
    Trino is a valuable platform to quickly analyze data. Many data analysts and data scientists 
    use Jupyter Notebooks consistently to write automated reports and schedule queries. 
    This makes Trino one of the most critical services to serve as a backbone of data access 
```

**Review by mosabua — CHANGES_REQUESTED** [review]
Text and such look good. Images need to be adjusted to the standard sizes 500x400 and 250x200 for the small.

**Comment by mosabua** [comment]
Looks like the images are now distorted. Can you fix that or am I mistaken. Also please sqaush all commits.

**Comment by mosabua** [comment]
@jatin5251 did you want us to take over this PR and get it completed?

**Comment by mosabua** [comment]
@jatin5251 let me know if I should create a replacement PR with the right logo sizes and such and get it merged.

**Comment by jatin5251** [comment]
@mosabua i will do it in a week, i am traveling right now. 
thanks for your patience

**Pull request closed without merging**